lib/tasks/check_samples_status.rake

```diff
@@ -32,12 +28,15 @@ task check_samples_status: :environment do
     mturk_batch_job.title = "#{mturk_batch_job.title} [#{hex[0..5]}]"
     mturk_batch_job.auto = true
     mturk_batch_job.job_file = project_batches[project.slug]
-    mturk_batch_job.mturk_worker_qualification_list_id = mturk_worker_qualification_list_id
+    mturk_batch_job.mturk_worker_qualification_list_id = primary_job.mturk_worker_qualification_list&.id
+    mturk_batch_job.max_tasks_per_worker = primary_job.max_tasks_per_worker
 
     mturk_auto_batch = MturkAutoBatch.new
     mturk_auto_batch.mturk_batch_job = mturk_batch_job
-    mturk_auto_batch.save!
+
     mturk_batch_job.save!
+    mturk_auto_batch.save!
```

Comment:
What does changing the order here accomplish? Would it make sense to wrap these in a transaction to guarantee all-or-nothing?